lib/ansible/modules/database/postgresql/postgresql_idx.py

```diff
@@ -1,173 +1,268 @@
 version_added: "2.8"
 options:
   idxname:
     description:
-      - Name of the index to create or drop.
+    - Name of the index to create or drop.
     type: str
     required: true
+    aliases:
+    - name
   db:
     description:
-      - Name of database where the index will be created/dropped.
+    - Name of database where the index will be created/dropped.
     type: str
   port:
     description:
-      - Database port to connect.
+    - Database port to connect.
     type: int
     default: 5432
   login_user:
     description:
-      - User (role) used to authenticate with PostgreSQL.
+    - User (role) used to authenticate with PostgreSQL.
     type: str
     default: postgres
+  session_role:
+    description:
+    - Switch to session_role after connecting.
+      The specified session_role must be a role that the current login_user is a member of.
+    - Permissions checking for SQL commands is carried out as though
+      the session_role were the one that had logged in originally.
+    type: str
+  schema:
+    description:
+    - Name of a database schema.
   login_password:
     description:
-      - Password used to authenticate with PostgreSQL.
+    - Password used to authenticate with PostgreSQL.
     type: str
   login_host:
     description:
-      - Host running PostgreSQL.
+    - Host running PostgreSQL.
     type: str
   login_unix_socket:
     description:
-      - Path to a Unix domain socket for local connections.
+    - Path to a Unix domain socket for local connections.
     type: str
   ssl_mode:
     description:
-      - Determines whether or with what priority a secure SSL TCP/IP connection
-        will be negotiated with the server.
-      - See U(https://www.postgresql.org/docs/current/static/libpq-ssl.html) for
-        more information on the modes.
-      - Default of C(prefer) matches libpq default.
+    - Determines whether or with what priority a secure SSL TCP/IP connection
+      will be negotiated with the server.
+    - See U(https://www.postgresql.org/docs/current/static/libpq-ssl.html) for
+      more information on the modes.
+    - Default of C(prefer) matches libpq default.
     type: str
     default: prefer
     choices: [ allow, disable, prefer, require, verify-ca, verify-full ]
   ssl_rootcert:
     description:
-      - Specifies the name of a file containing SSL certificate authority (CA)
-        certificate(s). If the file exists, the server's certificate will be
-        verified to be signed by one of these authorities.
+    - Specifies the name of a file containing SSL certificate authority (CA)
+      certificate(s). If the file exists, the server's certificate will be
+      verified to be signed by one of these authorities.
     type: str
   state:
     description:
-      - Index state.
+    - Index state.
     type: str
     default: present
-    choices: ["present", "absent"]
+    choices: [ present, absent ]
```

Comment:
Not a real issue, but we recommend to sort the order of lists if the order is not of importance.

Reply:
Good notice, done.